index.js

```diff
@@ -0,0 +1,210 @@
+const inquirer = require("inquirer");
```

Comment:
nice job importing your external packages at the top of the file! this really helps with readability, particularly in large projects :-)